server/data/data-jmap/src/test/java/org/apache/james/jmap/api/preview/MessagePreviewStoreContract.java

```diff
@@ -119,25 +120,37 @@ default void storeShouldBeIdempotent() {
     }
 
     @Test
-    default void concurrentStoreShouldStoreAllRecords() throws Exception {
+    default void concurrentStoreShouldStoreTheLatestValueDespiteOfCollisions() throws Exception {
```

Comment:
we don't want the latest, we want any value because all values should be identical